src/stream/stream_backend.rs

```diff
@@ -136,6 +144,23 @@ fn check_scheme(
                     return Err(SimpleError::new(format!("Endpoint with udp265 scheme only supports H265 encode. Encode: {:?}, Endpoints: {:#?}", encode, endpoints)));
                 }
             }
+            "webrtc" | "stun" | "turn" | "ws" => {
```

Comment:
We seriously need to handle that without the need of strings. We can do that in another PR.

Reply:
indeed, and we already have plans to do that: https://github.com/bluerobotics/mavlink-camera-manager/issues/50